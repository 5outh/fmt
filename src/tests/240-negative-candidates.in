<?php
//passes:MergeNamespaceWithOpenTag,NoSpaceAfterPHPDocBlocks,StripNewlineAfterCurlyOpen,StripNewlineAfterClassOpen,PrettyPrintDocBlocks,AlignTypehint,AlignDoubleSlashComments
$a . $b;
