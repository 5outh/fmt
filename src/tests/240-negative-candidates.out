<?php
//passes:MergeNamespaceWithOpenTag,NoSpaceAfterPHPDocBlocks,StripNewlineAfterCurlyOpen,StripNewlineAfterClassOpen,PrettyPrintDocBlocks,AlignTypehint,AlignDoubleSlashComments,GeneratePHPDoc,OrderMethod
$a . $b;
